app/serializers/trail_serializer.rb

```diff
@@ -0,0 +1,9 @@
+class TrailSerializer < ActiveModel::Serializer
+  attributes :id, :name, :street, :city, :state, :zip, :start_latitude, :start_longitude, :length, :difficulty, :description, :ascent, :photo, :low_elevation, :high_elevation
+
+  has_many :reviews
+
+  def reviews
+    object.reviews
+  end
```

Comment:
You shouldnt need this custom `reviews` method if you have line 4. This is redundant. Choose one or the other.

Reply:
Deleted method